News feed on the very top

VerevkinAlexander opened the pull request:
I have created a new component which includes dates filtration and news.json file and placed this component to the  top of page

VerevkinAlexander committed:
Added GitNews component and news file

VerevkinAlexander committed:
Added date verification

Commit:
show news

Signed-off-by: Stefan Schiessl <stefan.schiessl@blockchainprojectsbv.com>

Commit:
remove time

Signed-off-by: Stefan Schiessl <stefan.schiessl@blockchainprojectsbv.com>

Commit:
Merge branch 'develop' into news_feed_on_the_very_top

Commit:
add getNewsThroughAsset

Signed-off-by: Stefan Schiessl <stefan.schiessl@blockchainprojectsbv.com>

sschiessl-bcp commented:
Please 
- [x] add the possibility that the user closes the notification (persisted in localStorage), and if closed it won't open again (clickable "x" on the right side)
- [x] Please center the icon and text of the notification
- [x] If there are several notifications, only show one and have them switch after 15sec. Have the new notification slide in from the top

VerevkinAlexander committed:
-Added carousel
-Centered info Icon and text position

VerevkinAlexander commented:
I`ve added the new notification slide in from the top wich switched every 15 seconds and displayed only one notification. Also, I performed refactoring to improve component rendering. I`ve added new methods to the SettingsStore to get and persist hidenNews list in localStorage.Also, I`ve added the onClose method which added news to the hiddenNews list. Also, I added an additional close icon component to the Alert because of Alert`s onClose method with an included close icon closed several news notifications when we close one

VerevkinAlexander committed:
-Performed styling
- added get and set hiddenNews methods to LocalStoradge
- Added news filter
- Added carousel

sschiessl-bcp commented:
Accepted
